### ISSUE TITLE
ArgumentError on sublime.set_timeout

### ISSUE THREAD

**Issue by JohnEmhoff** [jisaacks/GitGutter#274]
Every run of GitGutter triggers the following exception:

```
Traceback (most recent call last):
  File "./sublime_plugin.py", line 205, in on_activated
  File "./sublime_plugin.py", line 154, in run_timed_function
  File "./sublime_plugin.py", line 204, in <lambda>
  File "./git_gutter_events.py", line 61, in on_activated
  File "./git_gutter_events.py", line 81, in debounce
Boost.Python.ArgumentError: Python argument types in
    sublime.set_timeout(function, NoneType)
did not match C++ signature:
    set_timeout(boost::python::api::object, int)
```

This is on Sublime 2, 2221.

**Comment by rchl** [comment]
GitGutter passes `settings.get("debounce_delay")` as a second argument. Which means that for you the setting doesn't exist.
Can you still reproduce this with latest version?

**Comment by rchl** [comment]
Closed as duplicate of #245.
